src/webapi/Controllers/EvaluationApiController.cs

```diff
@@ -0,0 +1,67 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using EdFi.OdsApi.Sdk.Apis.All;
+using EdFi.OdsApi.Sdk.Models.All;
+using Microsoft.AspNetCore.Mvc;
+using EdFi.OdsApi.Sdk.Client;
+using EdFi.OdsApi.SdkClient;
+using eppeta.webapi.Service;
+
+namespace webapi.Controllers
+{
+    [ApiController]
+    [Route("api/[controller]")]
+    public class EvaluationApiController : ControllerBase
+    {
+        private readonly IAuthenticationConfigurationService _service;
+        public EvaluationApiController(IAuthenticationConfigurationService service)
+        {
+            _service = service;
+        }
+
+        [HttpGet("configuration")]
+        public Configuration GetAuthenticatedConfiguration()
+        {
+            var authenticatedConfiguration = _service.GetAuthenticatedConfiguration();
+            return authenticatedConfiguration;
+        }
+
+        // GET: api/EvaluationApi
+        // The GetEvaluation method is slow due to the need to retrieve configuration first
+        [HttpGet]
+        public async Task<ActionResult<IEnumerable<TpdmEvaluationObjective>>> GetEvaluation()
+        {
+            try
+            {
+                // Get the authenticated configuration using the injected service
+                var authenticatedConfiguration = _service.GetAuthenticatedConfiguration();
+
+                // Get Evaluation Objectives
+                var apiInstance = new EvaluationObjectivesApi(authenticatedConfiguration);
+                apiInstance.Configuration.DefaultHeaders.Add("Content-Type", "application/json");
+
+                var evaluationObjectivesWithHttpInfo = apiInstance.GetEvaluationObjectivesWithHttpInfo(limit: 25, offset: 0, totalCount: true);
```

Comment:
What if there are more than 25? You don't have to solve for that immediately in this ticket - but if you _don't_ solve for it right now, then please create a ticket in Jira to "Add Paging Support in the Evaluation List", and then add a comment above this line, something like `// TODO: add paging support in {JIRA TICKET ID}`.

Reply:
After discussing further, it looks like paging may not be necessary because there aren't enough Evaluation Objectives to require it. Therefore ignore the comment above.

@stevenarnold-ed-fi please confirm; if you think that there will be more than 25 evaluation objectives at some point, then can you create another ticket to support getting more objectives?